bundles/distribution/pom.xml

```diff
@@ -538,41 +556,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>installer-test</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>integration-test</goal>
+                        </goals>
+                        <configuration>
+                            <environmentVariables>
+                                <ECLIPSELINK_UNZIP_DIR>${eclipselink.unzip.dir}</ECLIPSELINK_UNZIP_DIR>
+                            </environmentVariables>
+                            <includes>
+                                <include>org.eclipse.persistence.testing.jaxb.installer.InstallerTestSuite
+                                </include>
+                            </includes>
+                        </configuration>
+                    </execution>
```

Comment:
please add there 
```                    <execution>
                        <id>verify-integration-tests</id>
                        <goals>
                            <goal>verify</goal>
                        </goals>
                        <configuration>
                            <skip>${integration.test.skip.verify}</skip>
                        </configuration>
                    </execution>
```
to mark build result as a failure if installer-test will fail.

Reply:
done